platforms/browser/www/phonegap/plugin/facebookConnectPlugin/facebookConnectPlugin.js

```diff
@@ -161,16 +161,31 @@ cordova.define("com.phonegap.plugins.facebookconnect.FacebookConnectPlugin", fun
         
         // Bake in the JS SDK
         (function () {
+            // Retrieve the root element to append the script tags to. Use the body if no fb-root was found
+            var root = document.getElementById('fb-root') || document.getElementsByTagName('body')[0];
```

Comment:
Nice. This addition is really required. I cannot remember why Facebook used to insist on `fb-root` element.